recipes/libwebsockets/all/conanfile.py

```diff
@@ -419,13 +418,11 @@ def _configure_cmake(self):
         self._cmake.definitions["LWS_WITH_ALSA"] = False
         self._cmake.definitions["LWS_WITH_GTK"] = False
 
-
-
         self._cmake.configure()
         return self._cmake
 
     def _patch_sources(self):
-        if self.options.with_ssl != "False":
+        if tools.Version(self.version) == "4.0.15" and self.options.with_ssl != "False":
             tools.replace_in_file(
                 os.path.join(self._source_subfolder, "CMakeLists.txt"),
                 "list(APPEND LIB_LIST ws2_32.lib userenv.lib psapi.lib iphlpapi.lib)",
```

Comment:
While you're at this recipe, can you lowercase the `Crypt32.lib`?
mingw@linux is case sensitive.

Reply:
GitHub did let me fork to make a nice patch but I fixed the casing! If it fails I'll try again